platform/os_windows_test.go

```diff
@@ -116,34 +164,36 @@ func TestExecuteCommandError(t *testing.T) {
 }
 
 func TestSetSdnRemoteArpMacAddress_hnsNotEnabled(t *testing.T) {
-	mockExecClient := NewMockExecClient(false)
+	//mockExecClient := NewMockExecClient(false)
+	mockRegistry := initMockRegistry()
 	// testing skip setting SdnRemoteArpMacAddress when hns not enabled
-	mockExecClient.SetPowershellCommandResponder(func(_ string) (string, error) {
-		return "False", nil
-	})
-	err := SetSdnRemoteArpMacAddress(mockExecClient)
+	// mockExecClient.SetPowershellCommandResponder(func(_ string) (string, error) {
+	// 	return "False", nil
+	// })
```

Comment:
Feel free to remove these. Dead code can be found and resurrected with git if necessary.